### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by likeXXxx**

### CODE REVIEW

File: keps/sig-network/0011-ipvs-proxier.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ We will create a dummy interface and assign all kubernetes service ClusterIP's t
        valid_lft forever preferred_lft forever
 ```
 
-Note that the relationship between a Kubernetes service and an IPVS service is `1:N`. Consider a Kubernetes service that has more than one access IP. For example, an External IP type service has 2 access IP's (ClusterIP and External IP). Then the IPVS proxier will create 2 IPVS services - one for Cluster IP and the other one for External IP.
+Note that the relationship between a Kubernetes service and an IPVS service is `1:N`. Consider a Kubernetes service that has more than one access IP. For example, an External IP type service has 2 access IP's (ClusterIP and External IP). Then the IPVS proxies will create 2 IPVS services - one for Cluster IP and the other one for External IP.
```
Comment (cblecker) [inline keps/sig-network/0011-ipvs-proxier.md]: Please revert. This is intended.

**Comment by likeXXxx** [comment]
sorry,some things unexpected happend to my github account,I'll use my new github account to commit again

File: keps/sig-network/0011-ipvs-proxier.md (hunk 1 above)
Comment (likeXXxx) [reply keps/sig-network/0011-ipvs-proxier.md]: Yes,I've revert this.

**Pull request merged**